### PR TITLE
PEER-662 Add autotranslation feature

### PR DESCRIPTION
**Pull request by almondy22**

### CODE REVIEW

File: app/containers/ViewQuestion/Comments.jsx [hunk 1]
```diff
@@ -163,6 +168,13 @@ const CommentView = (item) => {
       history.comment?.id === `${item.postId}-${item.answerId}-${item.id}`,
   );
 
+  const getContent = () => {
+    if (+item.language === LANGUAGES_MAP[item.locale] || item.showOriginal) {
```
Comment (Lampardks) [inline app/containers/ViewQuestion/Comments.jsx:172]: Number(item.language)

File: app/containers/ViewQuestion/Comments.jsx [hunk 2]
```diff
@@ -189,6 +201,14 @@ const CommentView = (item) => {
         />
 
         <CommentManage>
+          <SeeOriginal
+            isOriginalLanguage={+item.language === LANGUAGES_MAP[item.locale]}
```
Comment (Lampardks) [inline app/containers/ViewQuestion/Comments.jsx:205]: Number(item.language)

File: app/containers/ViewQuestion/Content.jsx [hunk 3]
```diff
@@ -17,25 +17,50 @@ const BaseStyled = Base.extend`
 
 export const Content = (props) => {
   const isTemporaryAccount = false;
+  const [showOriginal, setShowOriginal] = useState(false);
+  const translation = props.translations?.find(
+    ({ language }) => +language === LANGUAGES_MAP[props.locale],
```
Comment (Lampardks) [inline app/containers/ViewQuestion/Content.jsx:22]: Number(language)

File: app/containers/ViewQuestion/Content.jsx [hunk 4]
```diff
@@ -17,25 +17,50 @@ const BaseStyled = Base.extend`
...
+  );
   //   !!props.author?.['integer_properties'].find(
   //   x => x.key === TEMPORARY_ACCOUNT_KEY && x.value,
   // );
+  const getContent = () => {
+    if (+props.language === LANGUAGES_MAP[props.locale] || showOriginal) {
```
Comment (Lampardks) [inline app/containers/ViewQuestion/Content.jsx:28]: Number(props.language)

File: app/containers/ViewQuestion/SeeOriginal.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+const SeeOriginal: React.FC<SeeOriginalProps> = ({
+  locale,
+  language,
+  showOriginal,
+  setShowOriginal,
+  translation,
+}): JSX.Element | null => {
+  const { t } = useTranslation();
+  const isOriginalLanguage =
+    +language === LANGUAGES_MAP[locale as keyof typeof LANGUAGES_MAP];
```
Comment (Lampardks) [inline app/containers/ViewQuestion/SeeOriginal.tsx:57]: Number(language)

File: app/containers/ViewQuestion/Comments.jsx (hunk 1 above)
Comment (almondy22) [reply app/containers/ViewQuestion/Comments.jsx]: Done

File: app/containers/ViewQuestion/Comments.jsx (hunk 2 above)
Comment (almondy22) [reply app/containers/ViewQuestion/Comments.jsx]: Done

File: app/containers/ViewQuestion/Content.jsx (hunk 4 above)
Comment (almondy22) [reply app/containers/ViewQuestion/Content.jsx]: Done

File: app/containers/ViewQuestion/SeeOriginal.tsx (hunk 5 above)
Comment (almondy22) [reply app/containers/ViewQuestion/SeeOriginal.tsx]: Done

**Review by Lampardks — APPROVED** [review]

**Pull request closed without merging**